### DISCUSSION TITLE
感觉最近绳网小助手更新的好频繁啊

### DISCUSSION THREAD

**Discussion by Kujojolen** [share121/inter-knot#1461] [其他]
大家伙察觉了吗？

**Comment by ZMJ2333** [comment]
e……yeah

**Comment by ZMJ2333** [comment]
What a coincidence, I also posted a similar post.

**Comment by GanlinSama** [comment]
好事啊，说明UP有在认真对待这个小天地（虽然一天更新三四次确实是有点烦哈）

**Comment by kallen1314** [comment]
今天格外频繁，虽然不知道具体更了什么，但想来作者可能一天都在忙这个，不然也不会那么多次更新了，作者辛苦，作者关心绳网，作者好

**Comment by gaojiaobei** [comment]
毕竟up一个人势单力薄，刚投入使用的绳网想要变成大伙期望的样子就只能靠up一点一点努力了
这真的是为爱发电

**Comment by GanlinSama** [comment]
上午来的时候还没有出等级
吃完晚饭再回来每个人头上都有个LV了
估计是在更新这个吧
（话说有没有大佬知道这个等级是根据什么划分的）

**Comment by Inter-Knot-User** [comment]
会楼上，这个委托我接了，github发布的项目数量

**Comment by qwq1949** [comment]
好事嘛~ .-.
